### PR TITLE
cling 1.0

### PR DESCRIPTION
**Pull request by clashman**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  cling
    * Files were found with references to the Homebrew shims directory.
      The offending files are:
        libexec/bin/cling
        libexec/lib/libclingJupyter.dylib
        libexec/lib/libcling.dylib
        libexec/lib/libclingInterpreter.a
```

**Comment by clashman** [comment]
> ```
>   cling
>     * Files were found with references to the Homebrew shims directory.
>       The offending files are:
>         libexec/bin/cling
>         libexec/lib/libclingJupyter.dylib
>         libexec/lib/libcling.dylib
>         libexec/lib/libclingInterpreter.a
> ```
Even after reading the source code of the function that throws the error I don't understand what the problem is. Is there any documentation anywhere?

Apparently fgrep is run on Homebrew's shim directory to look match for matches of the file names of the files contained in the keg? Please someone enlighten me or point me to the docs. I spent waay to much time on this already.

**Comment by chenrui333** [comment]
it is related to superenv https://docs.brew.sh/Formula-Cookbook#superenv-notes

File: Formula/c/cling.rb [hunk 1]
```diff
@@ -2,55 +2,36 @@ class Cling < Formula
...
-  bottle do
-    sha256               arm64_monterey: "ae9ec74f889a58e57f00394e3b46dd1793d975ce7dc5907c71e2e15853610a62"
-    sha256 cellar: :any, arm64_big_sur:  "82134eeea0ba90008355120b137908d828011e302b62ec97de10b152777d9651"
-    sha256               monterey:       "90f4150c5bcc027fe76db2f53948eb31e757124da337639303eee2ac768c8999"
-    sha256 cellar: :any, big_sur:        "e894d9476bc9ed0edb1ca8d3ca1d9fa6cefc8fc50befc93f1d1c25d1f1bee721"
-    sha256 cellar: :any, catalina:       "fd178b38640189a9b096d9c98fe3b1dedc934a504ddc0d3dc1c6bbfea144f09f"
-    sha256 cellar: :any, mojave:         "5135fc901ba316ca0e02f5598af21cd42a264994111252964f239b2576c7829b"
-    sha256               x86_64_linux:   "315073c45b0684a970493476b9c8476ddf90eb7d69bd5326efdf97b79ec55e25"
-  end
-
```
Comment (chenrui333) [inline Formula/c/cling.rb:18]: dont touch the bottle block

File: Formula/c/cling.rb [hunk 2]
```diff
@@ -2,55 +2,36 @@ class Cling < Formula
...
-  # Does not build on Ventura
```
Comment (chenrui333) [inline Formula/c/cling.rb:19]: the ventura build still failed?

**Pull request closed without merging**